daemon/mgr/cri_test.go

```diff
@@ -99,6 +172,37 @@ func TestToCriSandboxState(t *testing.T) {
 
 // Container related unit tests.
 
+func TestParseContainerName(t *testing.T) {
```

Comment:
Wrap all test cases into one loop, just like what you do in `TestFilterCRIContainers`, `TestFilterCRISandboxes`, etc.